rfcs/rdm-0080-bulk-importer.md

```diff
@@ -0,0 +1,104 @@
+Start Date: 2025-02-01
+- RFC PR: https://github.com/inveniosoftware/rfcs/pull/92
+- Authors: Aaron McCollough, Esteban G. Gabancho
+
+# Bulk importer
+
+## Summary
+
+Currently, InvenioRDM's only affordance for the bulk creation, import, and/or editing of records and files requires direct CLI-command-driven engagement with records and files APIs.
+
+GUI-based bulk importing and editing of records and files is a widely desired, highly useful feature, which will help to make the platform appealing to a much broader base of institutional users.
+
+The proposed feature is a beta version of a bulk importer for metadata (in CSV format) and associated files.
+
+## Motivation
+As an administrator user I want to ...
+- Upload many records (with their files) at once into my instance.
+- Select in which communities the records are publishes.
+- Determine if a group of records get new DOIs (minted during the process)
```

Comment:
is this use case already reflected in the mockup?

Reply:
Yes, on the third screen, there is a checkbox section. For now, it has two options: mint DOIs and publish.